docs/layouts/shortcodes/generated/checkpointing_configuration.html

```diff
@@ -18,7 +18,7 @@
             <td><h5>execution.checkpointing.checkpoints-after-tasks-finish</h5></td>
             <td style="word-wrap: break-word;">true</td>
             <td>Boolean</td>
-            <td>Feature toggle for enabling checkpointing even if some of tasks have finished. Before you enable it, please take a look at <a href="{{.Site.BaseURL}}{{.Site.LanguagePrefix}}/docs/dev/datastream/fault-tolerance/checkpointing/#checkpointing-with-parts-of-the-graph-finished-beta">the important considerations</a> </td>
+            <td>Feature toggle for enabling checkpointing even if some of tasks have finished. Before you enable it, please take a look at <a href="{{.Site.BaseURL}}{{.Site.LanguagePrefix}}/docs/dev/datastream/fault-tolerance/checkpointing/#checkpointing-with-parts-of-the-graph-finished">the important considerations</a> </td>
```

Comment:
Shouldn't it also be fixed in Chinese doc?

Reply:
yep, my bad, i noticed there is an occurrence in `docs/content.zh/release-notes/flink-1.15.md`  which is release related and shouldn't be touched.

Sorry for the false positive

Reply:
Thanks for the review and the conformation.